BDArmory/Ammo/RocketInfo.cs

```diff
@@ -120,9 +139,14 @@ public static void Load()
                             (bool)ParseField(node, "choker", typeof(bool)),
                             (bool)ParseField(node, "gravitic", typeof(bool)),
                             (bool)ParseField(node, "impulse", typeof(bool)),
+                            (float)ParseField(node, "massMod", typeof(float)),
+                            (float)ParseField(node, "force", typeof(float)),
```

Comment:
Ideally this would have just been switching the gravitic and impulse bools to floats and having them report false if <=0, done this way for backwards compatibility with mod rockets